jdbc/database_procedure/create_airline_insert_data.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE airline (al_name NVARCHAR(60), al_ccode NCHAR(2)) \g
```

Comment:
Are the demodb files packaged only with Ingres? I couldn't find the demodb files in my Vector 6.3 installation nor in the VW 6.3 documentation.

If that is the case, would it be appropriate to keep the script that creates and populates the table and update the README with clear instructions on using the demodb database if it exists, otherwise create the table using the included script?

Reply:
Interesting, it sounds like that is the case. In that case yes, mention Ingres demodb and then say use this cut-down schema if not available?

I'm assuming x100 nvarchar/nchar  DDL works fine?

Reply:
I tested creating the table using the clause `WITH STRUCTURE = X100` and the JDBC program was able to successfully call the procedure and return correct results.

I have also updated the README with instructions for using either the `demodb` database or a different database with the optional script to create the necessary table in PR #13